cluster-provision/k8s/1.24/manifests/cnao/operator.yaml

```diff
@@ -130,35 +130,36 @@ spec:
   template:
     metadata:
       annotations:
-        description: cluster-network-addons-operator manages the lifecycle of different Kubernetes network components on top of Kubernetes cluster
+        description: cluster-network-addons-operator manages the lifecycle of different
+          Kubernetes network components on top of Kubernetes cluster
```

Comment:
I guess that this is result of bumping cnao, so you might not be responsible for this, but shouldn't this be literal style, i.e a `|` in the end of L133?

Reply:
Yes its a result of bumping cnao[1] - I guess it should be fixed there as we are just consuming it here?

[1] https://github.com/kubevirt/cluster-network-addons-operator/blob/6130d3e299a658e75989c6f4cb81b805242b4664/manifests/cluster-network-addons/0.76.0/operator.yaml#L133

Reply:
Yes exactly.